filter.js

```diff
@@ -14,7 +14,8 @@ class Filter {
       valueNames: this.sortOptions.map(o => o.name)
     })
     /** Hide "open for both" filter */
-    this.$filters[2].parentElement.style.display = 'none'
+    // this.$filters[2].parentElement.style.display = 'none'
```

Comment:
Do we need this line?

Reply:
It's confusing to have a toggle for "open for both" since it's also captured in the open for donations/receiving filters. I think the way you addressed this works though